src/app/layout.tsx

```diff
@@ -16,7 +17,7 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className='inline-flex flex-col items-start'>{children}</body>
+      <body className="inline-flex flex-col items-start">{children}</body>
```

Comment:
why Switch to double here when ```metadata``` is using single?

Reply:
habit heh heh. We can stick with single.